tests/requirements.txt

```diff
@@ -13,3 +13,4 @@ docker
 better_exceptions
 pyasn1
 Jinja2
+httpx
```

Comment:
I think we probably ought to keep the pin, for stability: httpx == 0.21.*

Reply:
While this wasn't the case for our released Python package, I think it's OK if our tests break because an httpx upgrade broke compatibility?

Reply:
I would prefer if they didn't, and dependabot did the update on a PR instead. httpx is still pre-1.x, and a 0.N+1 can and does break (0.20 with allow_redirect for example).

On the LTS branch, I think it's definitely right to pin, and even on main I would prefer if we did.